src/com/latacora/backsaws/ini.clj

```diff
@@ -0,0 +1,40 @@
+(ns com.latacora.backsaws.ini
+  (:require
+   [clojure.java.io :as io]
+   [instaparse.core :as insta]
+   [babashka.fs :as fs]
+   [meander.epsilon :as m]))
+
+(def ini-parser (-> "ini.ebnf" io/resource insta/parser))
```

Comment:
nbd: I get why this is a noun, but personally, looking at how it’s called in the tests, in terms of API design I’d prefer it be a verb, e.g.

```diff
-  (def ini-parser (-> "ini.ebnf" io/resource insta/parser))
+  (def parse (-> "ini.ebnf" io/resource insta/parser))
```

and then it’d be called like `(ini/parse ini-string-from-file-or-whatever)`

Reply:
Hm, that's fair but note that it's not quite just a function:

```
(-> ini-parser :grammar :header :parsers)
;; =>
({:tag :string, :string "["}
 {:tag :nt, :keyword :wsp}
 {:tag :nt, :keyword :name}
 {:tag :nt, :keyword :wsp}
 {:tag :string, :string "]"}
 {:tag :nt, :keyword :wsp}
 {:tag :opt, :parser {:tag :nt, :keyword :comment}})
```

So as someone who understands instaparse, knowing that it's a parser and not an opaque fn (which might invoke the parser with specific arguments) is useful.

Reply:
Aha, very cool — thanks!

How about this then?

```suggestion
(def parser (-> "ini.ebnf" io/resource insta/parser))
(def parse parser)  ;; verb form for better readability at call sites
```

Reply:
Yup, done!

Reply:
Excellent!